service/src/main/kotlin/fi/espoo/evaka/invoicing/data/VoucherValueDecisionQueries.kt

```diff
@@ -342,6 +345,7 @@ fun Handle.searchValueDecisions(
                 AND youngest_child.area = ANY(:areas)
                 AND (:unit::uuid IS NULL OR part.placement_unit = :unit)
                 AND $freeTextQuery
+                ${if (financeDecisionManagerId != null) financeDecisionManagerWhere else ""}
```

Comment:
The idea nowadays is to use conditions like `AND (:unit::uuid IS NULL OR part.placement_unit = :unit)` with optional conditions so that the query string is as close to valid sql as possible. So in this case you could just write `AND (:financeDecisionManagerId::uuid IS NULL OR placement_unit.finance_decision_manager = :financeDecisionManagerId)`. The query in `FeeDecisionQueries.kt` is the old format.